kadi/tests/test_occweb.py

```diff
@@ -7,16 +8,14 @@
 # test collection by pytest.  Note that this does not work with
 # paramiko 2.0.0.
 from paramiko import py3compat
+import requests
 py3compat.u('dirty hack')
```

Comment:
Is this still needed?

Reply:
I took it out and test_occweb tests seems to pass. I'll push the commit and see if it flies in full integration testing.

Reply:
Done.